tests/regression/64-interval-set-one/test.py

```diff
@@ -0,0 +1,6 @@
+import os
+for index, filename in enumerate(os.listdir(".")):
+    if ".c" in filename:
+        new_filename = f"{index}{filename[2:]}"
+        #print(new_filename)
+        os.rename(filename, new_filename)
```

Comment:
I guess this was the script you used to make sure filenames are normalized. I don't think we should commit it though.